core/storage/suggestion/gae_models.py

```diff
@@ -229,6 +235,24 @@ def query_suggestions(cls, query_fields_and_values):
 
         return query.fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_translation_suggestions_with_exp_ids(cls, exp_ids):
+        """Gets all translation suggestions corresponding to explorations with
+        the given exploration ids.
+
+        Args:
+            exp_ids: list(str). list of exploration ids to query for.
+
+        Returns:
+            list(SuggestionModel). A list of translation suggestions that
+            correspond to the given exploration ids, up to a maximum of
+            feconf.DEFAULT_QUERY_LIMIT suggestions.
+        """
+        return cls.get_all().filter(
+            cls.suggestion_type == SUGGESTION_TYPE_TRANSLATE_CONTENT).filter(
+                cls.target_id.IN(exp_ids)).fetch(feconf.DEFAULT_QUERY_LIMIT)
```

Comment:
Nit: I think it'd be more readable if we split up the function calls onto their own lines somehow, e.g.:

```suggestion
        return (cls.get_all()
            .filter(cls.suggestion_type == SUGGESTION_TYPE_TRANSLATE_CONTENT)
            .filter(cls.target_id.IN(exp_ids))
            .fetch(feconf.DEFAULT_QUERY_LIMIT)
        )
```

Reply:
Done!